plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -205,6 +205,23 @@ void WriteSettings()
     DebugConfig::WriteSettings();
 }
 
+void CheckOutputModule(HWND window)
+{
+    OutputModule = SendMessage(GetDlgItem(window, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
+    const bool IsConfigurable =
+        mods[OutputModule] == PortaudioOut ||
+        mods[OutputModule] == WaveOut ||
+        mods[OutputModule] == DSoundOut;
+
+    const bool AudioExpansion =
+        mods[OutputModule] == XAudio2Out ||
+        mods[OutputModule] == PortaudioOut;
+
+    EnableWindow(GetDlgItem(window, IDC_OUTCONF), IsConfigurable);
+
```

Comment:
I'd prefer having this blank line removed.

Reply:
Sure.